Code/GraphMol/Depictor/testDepictor.cpp

```diff
@@ -20,6 +20,7 @@
 #include "RDDepictor.h"
 #include "DepictUtils.h"
 #include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/SmilesParse/SmilesWrite.h>
```

Comment:
Why is this change here? It doesn't look like anything else in this file has ben altered.

Reply:
I was probably writing out SMILES to test and forgot to remove this -- I will fix that :)